src/content/blog/2024_advisory_board_2025.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Introducing the Unitary Fund’s Advisory Board 2025
+author: Frances Poblete
+day: 21
+month: 11
+year: 2024
+tags: 
+  - advisory board
+  - members
+  - qiskit
```

Comment:
why the qiskit tag?

Reply:
@cosenal oh i might have just forgotten to remove this from where i copied it from :) Thanks!